### PR TITLE
Redesign layout

### PR DESCRIPTION
**Pull request by abbymapes**
I redesigned the whole layout of my portfolio and finished the map API and added authentication for comments. Sorry... this is a big one

### CODE REVIEW

File: portfolio/src/main/webapp/comments.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                    <div class="header"><h1>Comments</h1></div>
+                    <div class="comments">
+                        <div id = "comment-form">
+                          <!--<form action = "/data" method = "POST">
+                            <p style="font-size:14pt"><br> Leave a Comment: <br></p>
+                            <textarea name="name-input" placeholder="Write Name Here" id = "name-input"></textarea>
+                            <br><br>
+                            <textarea name="text-input" placeholder="Write Comment Here"></textarea>
+                            <br><br>
+                            <input type="submit"/>
```
Comment (dibenede) [inline portfolio/src/main/webapp/comments.html:38]: clean up commented out code

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    // User is logged in, so the request can proceed
+    String userEmail = userService.getCurrentUser().getEmail();
+    String logoutUrl = userService.createLogoutURL("/comments.html");
+    
+    String output = "";
+    output = "<p>You are logged in as: " + userEmail + "</p>";
+    output += "<p>To comment from a different email, <a href=\"" + logoutUrl + "\">logout</a>.</p>";
+    output += "Leave a display name and your comment below. Thanks for the feedback!</p>";
+    output += "<form action = \"/data?email="+ userEmail + "\" method = \"POST\">"+
+                "<p style=\"font-size:18pt\"><br> Leave a Comment: <br></p>"+
```
[the remark this answers is not in the document]
Comment (abbymapes) [reply portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java:61]: Thank you. I fixed this in next commit

File: portfolio/src/main/webapp/comments.html (hunk 1 above)
Comment (abbymapes) [reply portfolio/src/main/webapp/comments.html:38]: Fixed in next commit

File: portfolio/src/main/webapp/about.html [hunk 3]
```diff
@@ -64,39 +65,44 @@ <h3 style = "background-color: hsl(0, 0%, 100%, 0.5); padding:5px">Coding Langua
...
-                          <p><button onclick="document.location = 'artimages.html'">
-                                     Some of my Art</button></p>
-                          <p>Hiking, exploring, looking at, and traveling the world. </p>
-                          <p><button onclick="document.location = 'images.html'">
-                                     My Happy Places</button></p>
+                          <p style = "font-size: 14pt">Creating art.</p>
+                          <p><button onclick = "document.location = 'artimages.html'">Check it out</button>
+                        </div>
+
+                        <div class="sect-7">
```
Comment (dibenede) [inline portfolio/src/main/webapp/about.html:88]: you may want to give your sections more meaningful names. particularly since you have so many of them now

Comment (abbymapes) [reply portfolio/src/main/webapp/about.html]: Fixed this in next commit

File: walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java [hunk 4]
```diff
@@ -30,4 +30,22 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
+    Greeter greeter = new Greeter();
```
Comment (dibenede) [inline walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java:36]: while this exercise is pretty simple, you may want to give fixtures a try: https://junit.org/junit4/cookbook.html

**Pull request merged**